test/test.js

```diff
@@ -240,23 +242,60 @@ describe('Magister', function() {
 			})
 		})
 
-		it('should be able to create appointments and delete them', function () {
-			var now = new Date()
-			var today = new Date(now.getFullYear(), now.getMonth(), now.getDate())
-			var tomorrow = new Date(now.getFullYear(), now.getMonth(), now.getDate() + 1)
-			var nextWeek = new Date(now.getFullYear(), now.getMonth(), now.getDate() + 7)
+		it('should be able to create and delete appointments', function () {
+			const now = new Date()
+
+			const start = new Date(now.getFullYear(), now.getMonth(), now.getDate(), now.getHours())
+			const end = new Date(now.getFullYear(), now.getMonth(), now.getDate(), now.getHours() + 1)
+
+			const yesterday = new Date(now.getFullYear(), now.getMonth(), now.getDate() - 1)
+			const tomorrow = new Date(now.getFullYear(), now.getMonth(), now.getDate() + 1)
 
 			const description = 'magister.js test appointment'
 
 			return m.createAppointment({
-				start: today,
-				end: tomorrow,
-				description,
-			}).then(() => {
-				return m.appointments(today, nextWeek).then(r => {
-					const appointment = r.find(a => a.description === description)
-					expect(appointment).to.exist
-					return appointment.remove()
+				start: start,
+				end: end,
+				description: description,
+			}).then(response => {
+				expect(response.statusText).to.equal('Created')
+
+				return m.appointments(yesterday, tomorrow).then(appointments => {
+					const appointment = appointments.find(appointment => appointment.description === description)
+
+					return appointment.remove().then(response => {
+						// TODO: review: is this the most accurate way to test this?
+						expect(response.statusText).to.equal('No Content')
```

Comment:
Maybe this is also checkable with the status code?

Reply:
Well the `statusText` is just a text version of the `statusCode`, I think doing it this way is clearer (you instantly see that the status is `No Content` whereas `204` is something I'd have to look up to see what it means again) and should be as accurate. What inspired me to use this is the [`Django REST framework`](https://www.django-rest-framework.org/api-guide/status-codes/).

Reply:
Fair enough!